src/style.css

```
@@ -0,0 +1,30 @@
+body {
+  background-color: black;
+  margin: 10px;
+}
+
+.redish {
+  border: 1px solid #e75551;
+}
+
+.general {
```

as for me, better to write 
```suggestion
.outBox,  .midBox, .inBox{
  margin: 10px;
  border: 1px solid #e75551;
}
```